apps/contract/contracts/academy-progress-tracker/README.md

```
@@ -0,0 +1,69 @@
+## Building the Contract
+
+You can build the contract in two ways:
+
+1. From the contract directory:
+
+```bash
+cd contracts/academy-progress-tracker
+stellar contract build
+```
+
+2. From the project root:
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)